app-admin/logstash-bin/files/logstash-plugin.eselect

```diff
@@ -0,0 +1,77 @@
+# -*-eselect-*-  vim: ft=eselect
+# Copyright 2005-2017 Gentoo Foundation
+# Distributed under the terms of the GNU GPL version 2 or later
+
+DESCRIPTION="Manage user installed plugins of logstash"
+MAINTAINER="hydrapolic@gmail.com"
+
+PLUGINS="/var/lib/logstash/plugins-list.txt"
+LS_INSTALL_DIR="/opt/logstash"
+
+ls_plugin_wrapper() {
+  JARS_SKIP=true "${LS_INSTALL_DIR}/bin/logstash-plugin" "$1" "$2"
+}
+
+### add action
+describe_add() {
+  echo "Add plugin"
+}
+
+describe_add_parameters() {
+  echo "<plugin>"
+}
+
+do_add() {
+  local x
+  for x; do
+    echo "$x" >> ${PLUGINS} || die
+    ls_plugin_wrapper install "$x"
```

Comment:
What should happen if this fails? Maybe you should go for the install first, and die on failure, so that incorrect plugins are not recorded in the file.